.changes/v3.7.0/853-improvements.md

```diff
@@ -1,2 +1 @@
-* `resource/vcd_vm_sizing_policy`: remove (deprecate) unneeded `org` property [GH-853]
-* `datasource/vcd_vm_sizing_policy`: remove (deprecate) unneeded `org` property [GH-853]
+* `metadata` attribute on every compatible resource and data source is now more performant when adding and updating metadata [GH-853]
```

Comment:
> is now more performant

You mean refactoring increased speed? How come? :)

Reply:
Before we had a loop that iterated over all the metadata set. So imagine having a resource with a lot of metadata entries. Now, with the merge operation present in Go SDK is just one call, no looping required.

(Although for deletion we are still looping, as there is not an equivalent call for deleting all metadata at once)

Reply:
It's true that "more performant" can be too optimistic as I didn't check it empirically 😅